integration_tests/symbolics_05.py

```diff
@@ -21,4 +21,16 @@ def test_operations():
     print(a.diff(x))
     print(diff(b, x))
 
+    # test diff 2
+    c:S = sin(x)
+    d:S = cos(x)
+    assert(sin(Symbol("x")).diff(x) == d)
+    assert(sin(x).diff(Symbol("x")) == d)
+    assert(sin(x).diff(x) == d)
+    assert(sin(x).diff(x).diff(x) == S(-1)*c)
+    assert(sin(x).expand().diff(x).diff(x) == S(-1)*c)
+    assert((sin(x) + cos(x)).diff(x) == S(-1)*c + d)
+    assert((sin(x) + cos(x) + exp(x) + pi).diff(x).expand().diff(x) == exp(x) + S(-1)*c + S(-1)*d)
```

Comment:
Why don't we test this in the CI?

Reply:
I didn't catch you. Don't we execute `./run_tests.py -b=llvm_sym` and `./run_tests.py -b=c_sym` in `LPython CI / Run SymPy tests` ? I think so that checks this.

Reply:
Okay, cool.